Replace `pixi-react` PIXI package import with scoped packages (#391)
pixijs/pixi-react: packages/docs/docs/fallback-to-canvas.mdx

## Install

If using the entire pixi package, make sure to install the legacy package instead of `pixi.js`:

```bash
npm install pixi.js-legacy
```

> For advanced users using `@pixi` scoped modules, make sure to use the `canvas-` prefixed packages in place of the standard ones:

Reviewer: ```suggestion
For advanced users using `@pixi` scoped modules, make sure to properly import pixi's canvas packages:
```

> - `@pixi/display` - `@pixi/canvas-display`
> - `@pixi/extract` - `@pixi/canvas-extract`
> - `@pixi/graphics` - `@pixi/canvas-graphics`
> - `@pixi/mesh` - `@pixi/canvas-mesh`
> - `@pixi/particle-container` - `@pixi/canvas-particle-container`
> - `@pixi/prepare` - `@pixi/canvas-prepare`
> - `@pixi/renderer` - `@pixi/canvas-renderer`
> - `@pixi/sprite-tiling` - `@pixi/canvas-sprite-tiling`
> - `@pixi/sprite` - `@pixi/canvas-sprite`
> - `@pixi/text` - `@pixi/canvas-text`

Reviewer: ```suggestion
```ts
`import "@pixi/canvas-display"`
`import "@pixi/canvas-extract"`
`import "@pixi/canvas-graphics"`
`import "@pixi/canvas-mesh"`
`import "@pixi/canvas-particle-container"`
`import "@pixi/canvas-prepare"`
`import "@pixi/canvas-renderer"`
`import "@pixi/canvas-sprite-tiling"`
`import "@pixi/canvas-sprite"`
`import "@pixi/canvas-text"`
```ts

Also note that these will not be installed for you. You will need to install them yourself
```

Not sure if the double ```ts will be an issue or not but this is a more accurate description of what they need to do
